<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArduinoConsoleWindow</class>
 <widget class="QMainWindow" name="ArduinoConsoleWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Arduino Console</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPlainTextEdit" name="consoleOutput">
      <property name="toolTip">
       <string>Console area for showing text received from Arduino.</string>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonLayout"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="sliderLayout"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="ArduinoControlsLayout">
      <item>
       <widget class="QLabel" name="ArduinoPortLabel">
        <property name="text">
         <string>Arduino Port:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ArduinoPortName"/>
      </item>
      <item>
       <widget class="QPushButton" name="resetPlotButton">
        <property name="text">
         <string>Reset Plot</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ArduinoRawSwitch">
        <property name="text">
         <string>Show Raw Data</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ArduinoConnectSwitch">
        <property name="text">
         <string>Connected</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="commandLineLayout">
      <item>
       <widget class="QLabel" name="commandLineLabel">
        <property name="text">
         <string>Command Input:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="commandLine">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Command line for entering text to send to Arduino.</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>812</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTitle">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <addaction name="menuTitle"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ArduinoConnectSwitch</sender>
   <signal>toggled(bool)</signal>
   <receiver>ArduinoConsoleWindow</receiver>
   <slot>ArduinoConnectToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>758</x>
     <y>549</y>
    </hint>
    <hint type="destinationlabel">
     <x>793</x>
     <y>606</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ArduinoPortName</sender>
   <signal>returnPressed()</signal>
   <receiver>ArduinoConsoleWindow</receiver>
   <slot>ArduinoPortEntered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>556</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>607</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>commandLine</sender>
   <signal>returnPressed()</signal>
   <receiver>ArduinoConsoleWindow</receiver>
   <slot>commandEntered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>411</x>
     <y>579</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetPlotButton</sender>
   <signal>clicked()</signal>
   <receiver>ArduinoConsoleWindow</receiver>
   <slot>resetPlotPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>517</x>
     <y>549</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>607</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>commandEntered(QString)</slot>
  <slot>resetPressed()</slot>
  <slot>ArduinoConnectToggled(bool)</slot>
  <slot>ArduinoPortEntered()</slot>
  <slot>resetPlotPressed()</slot>
 </slots>
</ui>
